Client/src/Components/product-reviews/Helpful.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+// Passes in helpfulCOunter and incrementCounter. Allows user the option to click yes if the
+// review was helpful, and if it was it would incrememnt the helpfulCounter and remove the ability
+// to click yes again.
+function Helpful(props) {
+  const [isYes, setYes] = useState(false);
+  const { helpfulCounter } = props;
+  const { incrementCounter } = props;
+
+  function onClick() {
+    incrementCounter();
+    setYes(true);
+  }
+
+  function onHandleKeyDown() {
+    onClick();
+  }
+
+  if (!isYes) {
```

Comment:
Why not just conditionally render the "u" element?
For example: {!isYes  &&  (element)} //this only renders <u> if isYes is falsey.